src/main/java/org/dependencytrack/resources/v1/FindingResource.java

```diff
@@ -128,4 +136,42 @@ public Response exportFindingsByProject(@PathParam("uuid") String uuid) {
         }
     }
 
+    @POST
+    @Path("/project/{uuid}/analyze")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(
+            value = "Triggers Vulnerability Analysis on a specific project",
+            response = Project.class
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 401, message = "Unauthorized"),
+            @ApiResponse(code = 403, message = "Access to the specified project is forbidden"),
+            @ApiResponse(code = 404, message = "The project could not be found")
+    })
+    @PermissionRequired(Permissions.Constants.VIEW_VULNERABILITY)
+    public Response analyzeProject(
+            @ApiParam(value = "The UUID of the project to analyze", required = true)
+            @PathParam("uuid") String uuid) {
+        try (QueryManager qm = new QueryManager()) {
+            final Project project = qm.getObjectByUuid(Project.class, uuid);
+            if (project != null) {
+                if (qm.hasAccess(super.getPrincipal(), project)) {
+                  LOGGER.info("Analyzing Project " + project.getUuid());
```

Comment:
We probably want to log *who* triggered the analysis so abuse can be detected:

```suggestion
                  LOGGER.info("Analysis of project " + project.getUuid() + " requested by " + super.getPrincipal().getName());
```